static/templates/recipient_row.handlebars

```diff
@@ -9,13 +9,13 @@
                 title="{{#tr this}}Narrow to stream &quot;__display_recipient__&quot;{{/tr}}">
                 {{! invite only lock }}
                 {{#if invite_only}}
-                <i class="fa fa-lock invite-stream-icon" aria-hidden="true" title="{{t 'This is a private stream' }}"></i>
+                <i class="fa fa-lock invite-stream-icon" title="{{t 'This is a private stream' }}" aria-label="{{t 'This is a private stream' }}"></i>
```

Comment:
The information that it's a private stream isn't super high-priority information in the message feed context; does it make sense to tag it as ARIA or would it add clutter for the screenreader?